translations/en-us.yml

```diff
@@ -351,6 +352,7 @@ addons:
     configure:
         heading: 'Configure {providerName}'
         display-name: 'Display name'
+        go-to-root: 'Go to root'
```

Comment:
Should this be 'Go to home' or 'Go to home folder' to match up the aria label with the UI?